apps/api/v2/src/ee/schedules/controllers/schedules.controller.ts

```diff
@@ -27,81 +31,91 @@ import { CreateScheduleInput } from "../inputs/create-schedule.input";
 })
 @UseGuards(AccessTokenGuard)
 export class SchedulesController {
-  constructor(private readonly schedulesService: SchedulesService) {}
+  constructor(
+    private readonly schedulesService: SchedulesService,
+    private readonly schedulesResponseService: ResponseService
+  ) {}
 
   @Post("/")
   async createSchedule(
-    @GetUser("id") userId: number,
-    @Body() bodySchedule: CreateScheduleInput
-  ): Promise<ApiResponse<{ schedule: ScheduleResponse }>> {
-    const schedule = await this.schedulesService.createUserSchedule(userId, bodySchedule);
-    const scheduleResponse = schemaScheduleResponse.parse(schedule);
+    @GetUser() user: User,
+    @Body() bodySchedule: CreateScheduleInput,
+    @ForAtom() forAtom: boolean
+  ): Promise<ApiResponse<{ schedule: ScheduleResponse | ScheduleWithAvailabilitiesForWeb }>> {
+    const schedule = await this.schedulesService.createUserSchedule(user.id, bodySchedule);
+    const scheduleFormatted = await this.schedulesResponseService.formatSchedule(forAtom, user, schedule);
```

Comment:
NIT: wouldn't it be better if we name this as `formattedSchedules`?

Reply:
both are fine - if formatted is pre-fixed like you suggest or suffix like i have both are good